.github/workflows/link-check-on-pr.yml

```
@@ -42,7 +42,7 @@ jobs:
 
       - id: get-comment-body
         run: |
-          body="$(cat /tmp/lychee/out.md)"
+          body="$(echo "# Link check summary"; cat /tmp/lychee/out.md))"

```

There is one `)` too much here :)
```suggestion
          body="$(echo "# Link check summary"; cat /tmp/lychee/out.md)"
```